docs/project/library-servicing.md

```diff
@@ -29,9 +29,20 @@ If this library ships both inbox on a platform and in its own library package th
 ```
 Where the `AssemblyVersion` is set to the old version before updating. To determine if the library ships inbox you can look at for `InboxOnTargetFramework` item groups or `TreatAsOutOfBox` suppressions in the pkgproj for the library.
 
+If the library is part of a Aspnetcore or .NET targeting pack then we cannot increment the assembly version. For Aspnetcore, You can examine the ```<IsAspNetCoreApp>true</IsAspNetCoreApp>``` property in the library`s ```Directory.Build.Props```
```

Comment:
What about WindowsDesktop❔

Reply:
@ericstj who will be the best person to tell us on what packages are used by windowsDesktop in their targeting pack ?

Reply:
I don't know whom, but you could try spot checking them in the targeting pack installed in your dotnet SDK.